src/Filter.js

```diff
@@ -3,25 +3,14 @@ import './Filter.css';
 
 class Filter extends Component {
   render() {
+  let navBarItems = ['Population', 'GDP', 'Area', 'Climate', 'Continent'];
     return (
       <nav>
-        <li>
-          Population
-        </li>
-        <li>
-          GDP
-        </li>
-        <li>
-          Area
-        </li>
-        <li>
-          Climate
-        </li>
-        <li>
-          Continent
-        </li>
+        {navBarItems.map((item) => {
+                return <li> {item} </li>
+              }) }
       </nav>
```

Comment:
This was a great call cleaning up all of those <li> tags that were cluttering the file!

Reply:
I really love the layout and the color scheme. I do think the opacity for each card makes the card a little too distracting. Maybe we can lessen it, or think of another way to incorporate a modern look. Maybe drop shadow?